src/vector_transport.jl

```diff
@@ -51,45 +51,24 @@ compute ``Y = \operatorname{retr}_p^{-1}q``.
 # Constructor
 
     DifferentiatedRetractionVectorTransport(m::AbstractRetractionMethod)
-
-[^AbsilMahonySepulchre2008]:
-    > Absil, P.-A., Mahony, R. and Sepulchre R.,
-    > _Optimization Algorithms on Matrix Manifolds_
-    > Princeton University Press, 2008,
-    > doi: [10.1515/9781400830244](https://doi.org/10.1515/9781400830244)
-    > [open access](http://press.princeton.edu/chapters/absil/)
 """
 struct DifferentiatedRetractionVectorTransport{R<:AbstractRetractionMethod} <:
-       AbstractLinearVectorTransportMethod end
-function DifferentiatedRetractionVectorTransport(::R) where {R<:AbstractRetractionMethod}
-    return DifferentiatedRetractionVectorTransport{R}()
+       AbstractLinearVectorTransportMethod
+    retraction::R
+    function DifferentiatedRetractionVectorTransport(
+        r::R,
+    ) where {R<:AbstractRetractionMethod}
+        return new{R}(r)
+    end
```

Comment:
```suggestion
```
I think we don't need this constructor.

Reply:
Why? Would this work automatically?

Reply:
Yes, it looks exactly like the default one.